ledger/src/ui/test/mock/assert_utils.h

```diff
@@ -33,4 +33,5 @@
     assert(0 == strcmp((const char *)a, (const char *)b))
 #define ASSERT_STR_N_EQUALS(a, b, n) \
```

Comment:
Are we still using this one? We probably shouldn't if we are unless there's good reason to.